lib/spack/spack/build_systems/python.py

```diff
@@ -97,10 +97,11 @@ def configure(self, spec, prefix):
 
     extends('python')
 
-    def setup_file(self, spec, prefix):
+    def setup_file(self):
```

Comment:
Ah, I see. Since it's only a single file, I don't mind as much. But it is annoying to try to convert things to use `configure_args` and look out for whether or not they use `spec`. I think I'm more interested in convenience and you're more interested in maintaining a well-designed system. There are merits to both, obviously.

Leave the change in for now. You've convinced me, at least for this case.